cnvlib/diagram.py

```diff
@@ -39,6 +41,15 @@ def create_diagram(cnarr, segarr, threshold, min_probes, outfname, title=None):
         else:
             raise ValueError("Must provide argument cnarr or segarr, or both. ")
         do_both = False
+    
+    if show_range:
+        # QUESTION: Currently we ignore 'start', 'end' if any 
+        # Better to get them and raise error 'if not None' ??
```

Comment:
Thanks for adding this additional feature! — Yes, please check them and raise an error if not None, otherwise the behaviour might be confusing